### DISCUSSION TITLE
Design Notes - Instrument headers

### DISCUSSION THREAD

**Discussion by dkh-uci** [naccdata/uniform-data-set#61] [Forms] [answered]
Hi All,
For Form design, Instrument Headers asks for two text box fields:
![image](https://user-images.githubusercontent.com/101678603/216688354-bdc6f006-0260-4799-b628-63e50b08ca64.png)

However, github variable csv files do not include variable names for these 2 instrument header fields (B5 / B8).  
Should I just add the following text box as these variables:

formid_initials, ie b5_initials, b8_initials
formid_visitdate, ie, b5_visitdate , b8_visitdate

Also, does the date field need to be broken out by month , day , year to allow for unknown? 
b5_visitmo
b5_visitday
b5_visityr

**Comment by SuzanneHunt-KC** [comment, marked as answer]
@dkh-uci I had the same questions.  The specific instructions to add "a text box" made me think that we can use date validation and thus just 1 field instead of 3. @mcleod42

**Reply by SuzanneHunt-KC** [reply]
@mcleod42  Examiner’s initials has been set to alpha_only in NACC’s UDSv4 REDCap.  I don’t think that was previously required.  Perhaps this is what Michigan uses.  We have 3 employees with the same initials, including their middle names.  So I believe the newest employee has a number in their initials.  When uploading v2.0 and v3.0 data, the initials are uploaded as character which allows numbers to be used although they are not treated as a number.  I'm hoping to continue having that flexibility in UDS 4.0.  Are developers required to set the initials fields to be alpha-only ?

**Reply by mcleod42** [reply]
@SuzanneHunt-KC I confirmed with the Research Team that the Examiner's Initials text box field should remain un-validated. That gives folks the opportunity to enter both letters and numbers.

**Reply by mcleod42** [reply]
@dkh-uci I am working with the Research Team to get an answer for you on this. Thank you for your patience!

**Reply by mcleod42** [reply]
@dkh-uci @SuzanneHunt-KC my apologies. I apparently jumped the gun by putting the instructions about form visit date, examiner's initials and the footer on each form. The Research Team has not formally added those to the checklist. I will amend the Design Notes to indicate that those variable names are forthcoming. In the meantime, you are welcome to either add a placeholder variable or simply not include it. Once the Research Team has decided on those variable names, I will let you know. Again, my apologies for the miscommunication.

**Comment by mcleod42** [comment]
@SuzanneHunt-KC per the Research Team: previously when submitting UDS data, special characters were not allowed, as it would cause problems with SAS ingestion. However, future data submissions will not be in SAS, so there is no longer a restriction. This means you do not have to place any limitations or validations on 'specify' fields.

**Reply by haseenaR** [reply]
Thank you for this info.
